survive-the-monsters.js

```diff
@@ -38,6 +53,7 @@ scenes.main.preload = function() {
     this.load.image("player", "sprites/players/mvp_player/idle_0.png");
```

Comment:
**suggestion**: Consider adding gravity back to the physics configuration if future game mechanics require it. Currently, it's set to zero, which is fine for this implementation.

Reply:
it will be a top down shooter